docs/dashboards/chart-builder.md

```diff
@@ -0,0 +1,176 @@
+title: Chart Builder
+description: Sematext monitoring service exposes APIs and provides libraries that let you send custom metrics and turn them into custom charts and real-time visualizations
+
+Chart Builder enables you to create visualizations of the data sent to Sematext. It lets
+you build charts not only for metrics but also logs. Charts created through the builder
+are added to your reports or dashboards. Configuration of a chart requires specifying a list of
+series with the metrics, optional filters and groupings. Each series is translated to a query, sent to
+Sematext backend. Then extraction and processing of your data happens. Charts built through the 
+builder give you a great tool to show the spikes, trends helping you to understand how your
+application behaves in real time or how it behaved in the past.
+
+You can put many series on a single chart. Each can be configured to have it's own color and
+style (bar, line, area, points). Each series should be configured in a series
+configuration panel. The panel where you change the settings of each series is split in a
+couple sections.
+
+## Metrics
+
+In this section you pick one ore more metrics which are used to build a data series. If more
+than one metric is picked you have to apply transformations to let Chart Builder know what is
+the relation between metrics. If you don't write your own
+expression in transformation section all metrics are added to each other producing
+a single series.
+
+## Transformation
+
+Transformations are used to modify metrics with a formula. Why is this useful? Imagine you collect
+all the ingredients of cpu usage separately, like `os.cpu.user`, `os.cpu.system`, `os.cpu.wait`,
+... If you want to chart all the ingredients as a single series you have to
+sum them in an expression like the one below
+
+```
+`os.cpu.user` + `os.cpu.system` + `os.cpu.wait`
+```
+
+Another example is ratio. Imagine that you are interested in the ratio of the network input
+traffic and the network output traffic so you can easily spot suspicious behaviors of the system
+(sudden jump of output traffic which is not correlated with the input traffic jump). Then
+you simply divide one metric by another to get the interesting series:
+
+```
+`os.network.out` / `os.network.in`
+```
+
+_NOTE: All the metrics which have dot in the name have to be mentioned between_ ``` `` ``` .
+
+### Math operators
+
+You already know why transformations are needed. Here is what you can use to build transformations.
```

Comment:
You already know why transformations are needed. Here is what you can use to build transformations.
=>
Now you know why transformations are needed. Here is what you can use to build transformations.